design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,354 @@
+  over time as members need to be added or removed. For now the
+  workaround is to use terraform provider with [ignore_changes].
+
+In these cases, it would make sense to make Crossplane ignore select
+parameters during updates. Terraform addresses this use case with its
+[ignore_changes] functionality.
+
+An additional benefit of ignoring some fields would be that it would be
+possible to have different Crossplane Compositions managing a single
+external resource. This could open up some interesting new possibilities
+for Crossplane.
+
+The obvious way would be to just not set the field in the `spec.forProvider`.
+However, due to [Late Initialization], some fields are being set in
+`spec.forProvider` without any user input. Plus there are some cases
+where the field is required/wanted upon creation, but not on updates.
+
+So in this document, the aim will be to introduce a solution in Crossplane
+for ignoring some managed resource parameters during updates.
+
+## Goals
+
+* Design a functionality that allows for ignoring select fields during
+  resource updates.
+
+## Proposal
+
+Proposed solution is to use the new [managementPolicy] field which came with
+the [ObserveOnly] feature and add a new management policy that will 
+skip late initialization. The loss the information that the
+late initialization was providing would be offset by the `status.atProvider`
+which contains the resource state on the provider side and is being
+added alongside [ObserveOnly].
+
+In addition to the new management policy, we will also add a new field
+`spec.initProviders` which would contain parameters that should only be
+used at the creation time. This would allow users to specify fields that
+should be ignored on updates, but still be used during creation. This can
+be added as a separate feature.
+
+## Implementation
+
+### Management policy
+
+Proposed name for the new management policy is `PartialControl`, which
+would fit with the current `FullControl` and insinuate that the resource
+is not fully managed by Crossplane. Some other proposals for the naming
+are: `ExplicitControl`, `SelectiveControl`, `IgnoreLateInit`.
+
+The new management policy would be implemented in the `crossplane-runtime`
+and would just ignore the [ResourceLateInitialize] condition and 
+skip the managed resource update [here][Late Initialization Update] if
+the management policy is set to `PartialControl`.
+
+### Deletion policy
+
+`DeletionPolicy` was planned to be deprecated in favour of the new management
+policies according to [the ObserveOnly design doc.][ObserveOnly] However,
+with `PartialControl` it's not clear if it should delete or orphan the
+external resources. So to avoid creating `PartialControl` and
+`PartialControlOrphanOnDelete` management policies, we will keep the
+`DeletionPolicy` to be able to specify the behaviour. In that case, we will
+need to deprecate and remove the `OrphanOnDelete` management policy as it
+won't be needed anymore.
```

Comment:
Our alpha API contract allows us to just drop this enum value with no notice. We should definitely make it very obvious in the release notes, but we don't need a deprecation period.